providers/src/airflow/providers/cncf/kubernetes/kube_config.py

```diff
@@ -76,10 +76,6 @@ def __init__(self):
         # interact with cluster components.
         self.executor_namespace = conf.get(self.kubernetes_section, "namespace")
 
-        self.worker_pods_queued_check_interval = conf.getint(
```

Comment:
You might need to deprecate this config in the documentation. I'm not sure if it will handle this automatically